lib/model.js

```diff
@@ -572,7 +572,7 @@ module.exports = function(registry) {
         {arg: 'refresh', type: 'boolean', http: {source: 'query'}},
         {arg: 'options', type: 'object', http: 'optionsFromRequest'},
       ],
-      description: format('Fetches hasOne relation %s.', relationName),
+      description: g.f('Fetches hasOne relation %s.', relationName),
```

Comment:
I think `format` should be preserved?

Reply:
`format` is to format the string to take the parameters, which `g.f` can do. 
Here is an example: 
https://github.com/strongloop/generator-loopback/blob/master/oracle/index.js#L78